src/Controls/src/Core/IVisualElementController.cs

```diff
@@ -5,18 +5,40 @@
 
 namespace Microsoft.Maui.Controls
 {
+	/// <summary>For internal use by the <see cref="Controls"/> platform.</summary>
 	public interface IVisualElementController : IElementController
 	{
+		/// <summary>For internal use by the <see cref="Controls"/> platform.</summary>
```

Comment:
I'm not convinced that mdoc will translate Controls into Microsoft.Maui.Controls. Happy to be proven wrong though.

Reply:
I updated that cref to be  Microsoft.Maui.Controls